README.adoc

```diff
@@ -675,7 +675,7 @@ image:/images/git-checkout-to-specific-local-branch.png[Checkout to specific loc
 Branch name::
 
   If given, checkout the revision to build as HEAD on the named branch.
-  If value is an empty string or "**", then the branch name is computed from the remote branch without the origin.
+  If value is an empty string or "**", then the branch name is computed from the remote branch without the origin. If a specific revision and not branch HEAD is checked out, then 'detached' will be returned as the local branch name.
```

Comment:
Thanks for that addition.

Would you be willing to continue the existing formatting convention of a sentence per line so that documentation code reviews are easier for us?
```suggestion
  If value is an empty string or "**", then the branch name is computed from the remote branch without the origin.
  If a specific revision and not branch HEAD is checked out, then 'detached' will be returned as the local branch name.
```

Reply:
thanks. done